target/scripts/start-mailserver.sh

```diff
@@ -194,10 +194,16 @@ fi
 # marker to check if container was restarted
 date >/CONTAINER_START
 
+# Container logs will receive updates to this log file:
+MAIN_LOGFILE=/var/log/mail/mail.log
+# NOTE: rsyslogd would usually create this later during `_start_daemons`, however it would already exist if the container was restarted.
```

Comment:
I am confused about this comment. I don't get, what it tries to explain.

`touch` is used to avoid an error message from `wc`/`tail` if that file doesn't exist?

Reply:
Perhaps you have a better way to phrase it?

I wanted to make it clear that the `touch` is only necessary on first startup here to support `wc`/`tail` before `_start_daemons`, otherwise on a container restart it would already exist even without `touch` here as `rsyslogd` would have created when started.

Main intent was clarification that `touch` isn't the origin of this file being created, since it's implicitly created by `rsyslogd`, but we don't have a custom config maintained for that (_we kinda do, in the sense of `sed` in the `Dockerfile`_).

---

The offset could be 0 or 1 by default, and only update in the restart condition branch before calling `_start_daemons` if that would be a better approach?

I went with what we have here since it seemed nicer to group related lines, rather than add a little more complexity via branched logic 😅

Reply:
> Main intent was clarification that touch isn't the origin of this file being created

👍

> Perhaps you have a better way to phrase it?

I am totally fine with your detailed explanation. Now I get it 😄 

> I went with what we have here

That's fine 👍